<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataStatistics</class>
 <widget class="QWidget" name="DataStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>311</width>
     <height>54</height>
    </rect>
   </property>
   <property name="title">
    <string>统计模块1</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>课程数目：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Course">
      <property name="text">
       <string>None</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>学生数目：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Student">
      <property name="text">
       <string>None</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>631</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>统计模块2</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTableView" name="tableView_Course"/>
    </item>
    <item row="1" column="1">
     <widget class="QTableView" name="tableView_Student"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string> 所有课程（请选择课程查看选修该课程的学生）</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>         选择该课程的学生</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_Fresh">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新统计数据</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
